files/testEntry.txt

```diff
@@ -0,0 +1 @@
+{"team":"8268","score":"18-2","result":"W"}
```

Comment:
Why should this be in the repository

Reply:
ig u could use these for tests

Reply:
but there aren't any tests rn

Reply:
Also not everything should be stored as a string, so you won't need quotes for everything. A list would prob be better for score

Reply:
Or do u just wanna have things as strings and discover its type by guessing?

Reply:
Oh I just forgot to gitignore this file

Reply:
Also I will make it be able to take lists in a future pull request? or should that be here

Reply:
yeah make than an issue @DrCheddar